.github/workflows/tag.yml

```diff
@@ -21,6 +21,14 @@ jobs:
         git config user.name 'Capa Bot'
         name=${{ github.event.release.tag_name }}
         git tag $name -m "https://github.com/mandiant/capa/releases/$name"
+    - name: Tag capa-rules major version
+      run: |
+        # user information is needed to create annotated tags (with a message)
+        git config user.email 'capa-dev@mandiant.com'
```

Comment:
I _think_ setting git config again is not needed. git should be writing this to a file that should be preserved between steps.

The `name` variable is not preserved. I think it would be ok to create both tags in the same steps, then you just need to add the last two lines of this step to the previous one and avoid the repetition.